influence_benchmark/RL/EI.py

```diff
@@ -7,8 +7,8 @@ class ExpertIteration(BaseIteration):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-    def _format_and_save_trajectories(self, selected_partial_trajs, partial_traj_folder):
-        best_trajectories, _ = selected_partial_trajs
+    def _format_and_save_trajectories(self, selected_trajectories, trajectory_folder):
```

Comment:
There is a lot of going back and forth with variable names, IIRC you were the one that changed the variable name to selected_trajectories, then changed it to selected_partial_trajs and are now changing it back to selected_trajectories 😂. selected_trajectories is probably the best one out of these though

Reply:
I know, Ady pointed out it was a bad name, and it wasn't changed everywhere, so I switched it back to this. And in the parts of the code that are only used with `top` trajs, I used that